CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.3.2 (2020-04-14)
+
+- Fixed the colorpicker component to hide the launch and reset buttons when the colorpicker's input element is set to a type of "hidden". [#40](https://github.com/blackbaud/skyux-colorpicker/pull/40)
```

Comment:
Could we tighten up to "Fixed the colorpicker to hide the launch and reset buttons when the input element's `type` is `hidden`."?